tests/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+	"compilerOptions": {
+		"moduleResolution": "node",
+		"module": "ESNext",
+		"target": "es2022",
+		"lib": ["es2022", "dom"],
+		"types": [
+			"node",
+			"@wdio/globals/types",
+			"@wdio/mocha-framework",
+			"webdriverio/async",
+			"expect-webdriverio",
+			"webdriverio/async",
```

Comment:
duplicated item

Reply:
also seems to be redundant
https://stackoverflow.com/questions/75057693/cannot-find-type-definition-file-for-wdio-globals-types